build/infrastructure/main/st-marketoplogs.tf

```
@@ -0,0 +1,56 @@
+# Copyright 2020 Energinet DataHub A/S
+#
+# Licensed under the Apache License, Version 2.0 (the "License2");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+locals {
+  marketoplogs_container_name = "marketoplogs"
+}
+
+module "st_market_operator_logs" {
+  source                    = "git::https://github.com/Energinet-DataHub/geh-terraform-modules.git//azure/storage-account?ref=5.1.0"
+
+  name                      = "marketoplogs"
+  project_name              = var.domain_name_short
+  environment_short         = var.environment_short
+  environment_instance      = var.environment_instance
+  resource_group_name       = azurerm_resource_group.this.name
+  location                  = azurerm_resource_group.this.location
+  account_replication_type  = "LRS"
+  access_tier               = "Hot"
+  account_tier              = "Standard"
+  containers                = [
+    {
+      name = local.marketoplogs_container_name,
+    },
+  ]
+
+  tags                      = azurerm_resource_group.this.tags
+}
+
+module "kvs_st_market_operator_logs_primary_connection_string" {
+  source        = "git::https://github.com/Energinet-DataHub/geh-terraform-modules.git//azure/key-vault-secret?ref=5.1.0"
+  name          = "st-marketoplogs-primary-connection-string"
```

Add empty line between source and name

```suggestion
  source        = "git::https://github.com/Energinet-DataHub/geh-terraform-modules.git//azure/key-vault-secret?ref=5.1.0"
  
  name          = "st-marketoplogs-primary-connection-string"
```